doc/man7/EVP_PKEY-EC.pod

```diff
@@ -187,6 +187,28 @@ that m > k3 > k2 > k1 > 0
 
 =back
 
+The following KeyGen settable parameter is also available for the OpenSSL FIPS
```

Comment:
_key generation_ instead of _KeyGen_.

There is another instance of this on line 22 which could be updated too.